base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -1400,24 +1400,21 @@ impl LMDBDatabase {
     ) -> Result<(), ChainStorageError> {
         let store = self.validator_node_store(txn);
         let constants = self.get_consensus_constants(header.height);
-        let current_epoch = constants.block_height_to_epoch(header.height);
-
-        let prev_shard_key = store.get_shard_key(
-            current_epoch
-                .as_u64()
-                .saturating_sub(constants.validator_node_validity_period_epochs().as_u64()) *
-                constants.epoch_length(),
-            current_epoch.as_u64() * constants.epoch_length(),
-            vn_reg.public_key(),
-        )?;
+        let current_epoch = self.block_height_to_epoch(header.height);
+        // TODO: What if the validity period has changed?
```

Comment:
```suggestion
```
If this is important enough create an issue, a todo randomly in code will just get deleted after a while anyway.